modules/mssql/versions.tf

```diff
@@ -20,7 +20,7 @@ terraform {
 
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = "~> 3.60"
+      version = "~> 4.8.0"
```

Comment:
Thanks for the PR @dimitri-fert 
I merged the other PR as we wanted to continue supporting 3.x

Reply:
Thanks for this. Appreciated :+1: